src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataResourceSetWrapperBase.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.OData;
+
+namespace Microsoft.AspNet.OData.Formatter.Deserialization
+{
+    /// <summary>
+    /// Encapsulates an <see cref="ODataResourceSet"/> and the <see cref="ODataResource"/>'s that are part of it.
+    /// </summary>
+    public abstract class ODataResourceSetWrapperBase : ODataItemBase
```

Comment:
> public abstract class ODataResourceSetWrapperBase [](start = 3, length = 50)

I can't understand why should we have the ODataResourceSetWrapperBase base class?

But, for others, 
1) ODataResourceWrapper
2) ODataDeltaLinkWrapper

There's no ....Base.cs class?